js/repl/CodeMirrorPanel.js

```diff
@@ -70,13 +70,15 @@ const styles = {
     backgroundColor: colors.errorBackground,
     borderTop: `1px solid ${colors.errorBorder}`,
     color: colors.errorForeground,
+    fontFamily: "monospace",
```

Comment:
We can add this to `sharedBoxStyles` to avoid needing to add it twice.

Reply:
👍 I didn't notice that `sharedBoxStyles` is only used for these two boxes.